control-plane/cmd/kafka-controller/main.go

```diff
@@ -76,22 +76,21 @@ func main() {
 			},
 		},
 
-		// TODO: enable later
-		//// Namespaced broker controller
-		//injection.NamedControllerConstructor{
-		//	Name: "broker-namespaced-controller",
-		//	ControllerConstructor: func(ctx context.Context, watcher configmap.Watcher) *controller.Impl {
-		//		return broker.NewNamespacedController(ctx, watcher, brokerEnv)
-		//	},
-		//},
-		//
-		//// Namespaced trigger controller
-		//injection.NamedControllerConstructor{
-		//	Name: "trigger-namespaced-controller",
-		//	ControllerConstructor: func(ctx context.Context, watcher configmap.Watcher) *controller.Impl {
-		//		return trigger.NewNamespacedController(ctx, watcher, brokerEnv)
-		//	},
-		//},
+		// Namespaced broker controller
+		injection.NamedControllerConstructor{
+			Name: "broker-namespaced-controller",
```

Comment:
nit: how about `namespaced-broker-controller` (and `namespaced-trigger-controller`)?

Reply:
Sounds good. I want to get this one merged now. I can change this and also @pierDipi 's suggestion in a separate PR